src/yoroi-wallets/cardano/ShelleyWallet.ts

```diff
@@ -7,8 +7,10 @@ import cryptoRandomString from 'crypto-random-string'
 import ExtendableError from 'es6-error'
```

Comment:
should this class be in `src/yoroi-wallets/cardano/shelley`?

Reply:
probably be renamed to CardanoWallet, its not really just shelley though